package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "test": "./node_modules/karma/bin/karma start test/config/karma.conf.js",
     "build:dev": "webpack",
-    "build:prod": "webpack --env=prod",
```

Comment:
Don't remove this line or other dependencies in package.json :D What Huy meant is that we should remove the redundant plugin declaration in webpack.test.js only, not remove them in package.json. Because they are used in webpack.prod.js (when we build the production version)